src/test/java/com/exasol/adapter/document/DocumentAdapterIT.java

```diff
@@ -292,13 +282,66 @@ void testInlineMappingArray() throws SQLException {
                 .mapping(mapping).build();
         final EdmlSerializer edmlSerializer = new EdmlSerializer();
         final String mappingString = "[" + edmlSerializer.serialize(t1) + ", " + edmlSerializer.serialize(t2) + "]";
-        final VirtualSchema virtualSchema = createVirtualSchema(mappingString);
-        try (final Statement statement = connection.createStatement()) {
-            final ResultSet resultSet = statement.executeQuery(
-                    "SELECT * FROM " + MY_VIRTUAL_SCHEMA + ".T1 UNION ALL SELECT * FROM " + MY_VIRTUAL_SCHEMA + ".T2");
-            assertThat(resultSet, table().row("123456789").row("123456789").matches(NO_JAVA_TYPE_CHECK));
-        } finally {
-            virtualSchema.drop();
+        try (final VirtualSchema virtualSchema = createVirtualSchema(mappingString)) {
+            assertQueryResult(
+                    "SELECT * FROM " + MY_VIRTUAL_SCHEMA + ".T1 UNION ALL SELECT * FROM " + MY_VIRTUAL_SCHEMA + ".T2",
+                    table().row("123456789").row("123456789").matches(NO_JAVA_TYPE_CHECK));
+        }
+    }
+
+    @Test
+    void testSetProperty() throws SQLException {
+        final EdmlDefinitionBuilder builder = EdmlDefinition.builder().source("")//
+                .destinationTable("BOOKS");
+        final EdmlDefinition mapping1 = builder.mapping(Fields.builder()//
```

Comment:
Shouldn't there be some more cases? — E.g. 
* leave field unchanged :white_check_mark: 
* add field :white_check_mark: 
* remove field :x: 
* change data type of field :x: